components/tab-switcher.jsx

```diff
@@ -0,0 +1,81 @@
+// Note: source code of this component is pretty much copied directly from https://github.com/mozilla/science.mozilla.org
```

Comment:
@alanmoo 

solved outdated `node` problem but nothing much we can do for `react` and `react-router` 😢 

```bash
npm WARN mofo-ui@1.6.0 requires a peer of react@^0.14.0 || ^15.0.0 but none was installed.
npm WARN react-router-scroll@0.3.2 requires a peer of react@^0.14.0 || ^15.0.0 but none was installed.
npm WARN react-router-scroll@0.3.2 requires a peer of react-router@^2.3.0 but none was installed.
```